providers/implementations/asymciphers/rsa_enc.c

```diff
@@ -110,6 +110,10 @@ static int rsa_init(void *vprsactx, void *vrsa, const OSSL_PARAM params[],
 
     switch (RSA_test_flags(prsactx->rsa, RSA_FLAG_TYPE_MASK)) {
     case RSA_FLAG_TYPE_RSA:
+        if (!ossl_prov_has_pkcs1_v15_padding(prsactx->libctx)) {
+            ERR_raise(ERR_LIB_PROV, PROV_R_PKCS1_15_PADDING_UNSUPPORTED);
+            return 0;
+        }
```

Comment:
This is just setting the default - it's bad to block it here because the padding can be later set to OAEP padding.

Reply:
It should be blocked in rsa_encrypt() call.

Reply:
Or change the default to OAEP padding if v1.5 is not allowed?